bdai_ros2_wrappers/bdai_ros2_wrappers/static_transform_broadcaster.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2023 Boston Dynamics AI Institute Inc.  All rights reserved.
+from typing import Dict, Iterable, Optional, Union, cast
+
+from geometry_msgs.msg import TransformStamped
+from rclpy.node import Node
+from rclpy.qos import DurabilityPolicy, HistoryPolicy, QoSProfile
+from tf2_msgs.msg import TFMessage
+
+
+class StaticTransformBroadcaster:
+    """
+    A modified :class:`tf2_ros.StaticTransformBroadcaster` that stores transforms
+    sent through it, matching ``rclcpp::StaticTransformBroadcaster`` behavior.
+    """
+
+    DEFAULT_QOS = QoSProfile(depth=1, durability=DurabilityPolicy.TRANSIENT_LOCAL, history=HistoryPolicy.KEEP_LAST)
+
+    def __init__(self, node: Node, qos: Optional[Union[QoSProfile, int]] = None) -> None:
+        """
+        Constructor.
+
+        :param node: The ROS2 node.
+        :param qos: A QoSProfile or a history depth to apply to the publisher.
+        """
+        if qos is None:
+            qos = StaticTransformBroadcaster.DEFAULT_QOS
+        self._net_transforms: Dict[str, TransformStamped] = {}
+        self._pub = node.create_publisher(TFMessage, "/tf_static", qos)
+
+    def sendTransform(self, transform: Union[TransformStamped, Iterable[TransformStamped]]) -> None:
+        try:
+            transforms = list(transform)
+        except TypeError:
```

Comment:
When does this exception happen? If it's something fairly important or regular, I wouldn't mind a little comment on when and why this happens. It'll help for future maintainability so someone doesn't have rediscover that error exception pathway.

Reply:
See https://github.com/bdaiinstitute/ros_utilities/pull/36/commits/88a11a609ee3764533e13b0502e4281bc5a696c4.